website/docs/r/virtual_network_gateway.html.markdown

```diff
@@ -186,7 +186,7 @@ The `vpn_client_configuration` block supports:
     This setting is incompatible with the use of `root_certificate` and `revoked_certificate`.
 
 * `vpn_client_protocols` - (Optional) List of the protocols supported by the vpn client.
-    The supported values are `SSTP` and `IkeV2`.
+    The supported values are `SSTP`, `IkeV2` and `OpenVPN`. **NOTE**: The OpenVPN protocol is currently in Public Preview. You need to [opt in to be able to use OpenVPN as an option](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-openvpn).
```

Comment:
for `NOTE` boxes we use a separate style for this (a blue/yellow box on it's own line) - as such could we make this:

```
-> **NOTE:** Support for `OpenVPN` as a Client Protocol is currently in Public Preview - [you can register for this Preview using this link](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-howto-openvpn).
```

(I've also reworded this slightly to match the language used in our other boxes)

Reply:
Sure, I updated the doc with your suggestion.